src/MongoDB.Driver.Core/Core/Operations/AggregateToCollectionOperation.cs

```diff
@@ -213,6 +229,9 @@ internal BsonDocument CreateCommand(ICoreSessionHandle session, ConnectionDescri
             var serverVersion = connectionDescription.ServerVersion;
             Feature.Collation.ThrowIfNotSupported(serverVersion, _collation);
 
+            var readConcern = _readConcern != null
```

Comment:
`GetWriteConcernForCommandThatWrites` has checking `_writeConcern` on null inside the method.